code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -485,3 +485,168 @@
 		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right)
 		)
 
+///////////////////////////
+/////////KHARMAANI/////////
+///////////////////////////
+/datum/species/mantid //Only Alates for now
+//Features from bay not ported (yet???): Gynes (Big Mantids), Crystalline Limb mechanics, Razorweb Weaving, and assorted Mantid gear that probably isn't needed here.
+
+	name =                   SPECIES_MANTID_ALATE
+	name_plural =            "Kharmaan Alates"
+	show_ssd =               "quiescent"
+	language =         LANGUAGE_MANTID_NONVOCAL
+	default_language = LANGUAGE_MANTID_VOCAL
+	secondary_langs =  list(LANGUAGE_GALCOM)
+	num_alternate_languages = 2
+
+	blurb = "Relatively new to the Periphary, the Kharmaani are a race of reclusive crystalline mantids. \
+	Kharmaan Alates are the most common and currently only face of the Ascent in Virgo-Erigone. With a \
+	culture of uncompromising servitude beaten into them with bladed forelimbs and a biology prone to \
+	eusociality, many Alates have found the relative isolation from the domineering larger castes and the \
+	drive to follow orders to be freeing, and have proven themselves remarkably sociable and varied in \
+	personality and drive, if a bit eccentric."
+
+	icobase =                 'icons/mob/human_races/species/ascent/alate/body.dmi'
+	deform =                  'icons/mob/human_races/species/ascent/alate/body.dmi'
+	damage_overlays =         'icons/mob/human_races/species/ascent/alate/damage_mask.dmi'
+	blood_mask =              'icons/mob/human_races/species/ascent/alate/blood_mask.dmi'
+	//organs_icon =             'icons/mob/human_races/species/ascent/organs.dmi'
+	//I couldn't get the custom organ names and sprites to work here, someone else is free to give it a shot.
+
+	blood_color =             "#660066"
+	flesh_color =             "#009999"
+	hud_type =                /datum/hud_data/mantid
+
+	speech_chance = 100
+	speech_sounds = list(
+		'sound/voice/ascent1.ogg',
+		'sound/voice/ascent2.ogg',
+		'sound/voice/ascent3.ogg',
+		'sound/voice/ascent4.ogg',
+		'sound/voice/ascent5.ogg',
+		'sound/voice/ascent6.ogg'
+	)
+
+	siemens_coefficient =   0.2 // Crystalline body.
+	oxy_mod =               0.8 // Don't need as much breathable gas as humans.
+	toxins_mod =            0.4 // Not as biologically fragile as meatboys.
+	radiation_mod =         0.4 // Not as biologically fragile as meatboys.
+	flash_mod =               2 // Highly photosensitive.
+	brute_mod =             1.5 // Incredibly fragile.
+	burn_mod =              0.5 // Yet reflective.
+	darksight = 7
+
+	min_age =                18
+	max_age =                140 //
+	slowdown =               -0.2
+	rarity_value =            3
+	gluttonous =              2
```

Comment:
Remove gluttonous, since we do not use it in our codebase.

Reply:
Fixed